doc/api/http.md

```diff
@@ -1146,8 +1146,12 @@ This method signals to the server that all of the response headers and body
 have been sent; that server should consider this message complete.
 The method, `response.end()`, MUST be called on each response.
 
-If `data` is specified, it is equivalent to calling
-[`response.write(data, encoding)`][] followed by `response.end(callback)`.
+If `data` is specified, it is functionally equivalent to calling
+[`response.write(data, encoding)`][] followed by `response.end(callback)`. There
+is, however, a difference in the headers that are sent by default. If
+[`response.write()`][] is used, the `Transfer-Encoding` header is set to
+`chunked`, otherwise the `Transfer-Encoding` header is replaced by the
```

Comment:
Nit:
```suggestion
`'chunked'`, otherwise the `Transfer-Encoding` header is replaced by the
```

Reply:
Optional suggestion: Change the comma to a period and make the `Otherwise` clause its own sentence.